apertium-get.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python3
+
+import requests, argparse, subprocess, os, re, sys
```

Comment:
pep8: imports usually on their own lines.

Reply:
Can we avoid the `requests` dependency? Let's stick with only stdlib deps.

Reply:
Will do - I kind of forgot that `requests` wasn't standard library.